src/animation.jl

```diff
@@ -22,6 +22,7 @@ giffn() = (isijulia() ? "tmp.gif" : tempname() * ".gif")
 movfn() = (isijulia() ? "tmp.mov" : tempname() * ".mov")
 mp4fn() = (isijulia() ? "tmp.mp4" : tempname() * ".mp4")
 webmfn() = (isijulia() ? "tmp.webm" : tempname() * ".webm")
+apngfn() = (isijulia() ? "tmp.png" : tempname() * ".png")
```

Comment:
Why the `.png` extension over `.apng` ?

Reply:
As i saw during my researches, APNG files commonly use the standard ".png" extension. Nevertheless the function should be named apng for consistency.

Reply:
According to https://en.wikipedia.org/wiki/APNG, both `.png` and `.apng` are possible, so I'd rather use the dedicated `.apng` for animations (better be explicit than implicit, consistent with the function name `apngfn`) unless there is an argument against that ?

Reply:
I agree with you, i just believed that people used .png for both comonly. But using `.apng` makes more sense obvisouly. I change that

Reply:
Well, i tested this change. `.apng` files are not recognized by vscode, while `.png` animated files are.

Reply:
https://github.com/microsoft/vscode/blob/main/src/vs/base/common/mime.ts#L70: `png`, `gif`, `webp` but no `.apng`. I don't use vscode, so If that doesn't work, I'd suggest reverting to `.png` and add an explicit note to explain why we are forced to use `.png` over `.apng` for compat with vscode.

Reply:
@curio-sitas, in the end, is this solved for `vscode` ?